app/controllers/choose_role_controller.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
 class ChooseRoleController < ApplicationController
-  def index; end
+  layout 'initial_application'
+
+  def index
+    redirect_to my_account_path if user_signed_in?
```

Comment:
If the signed in user is kid, should not we redirect to the shared account instead?

Reply:
```
class MyAccountsController < ApplicationController
  before_action :authenticate_user!

  def show
    redirect_to account_path(shared_accounts.first) unless current_user.parent?
  end
  
  ***
end
```